api/src/controllers/elasticsearch/dashboard/sejour.js

```diff
@@ -0,0 +1,252 @@
+        },
+        precense: {
+          ...aggsFilter,
+          aggs: { names: { terms: { field: "cohesionStayPresence.keyword", missing: "NR", size: ES_NO_LIMIT } } },
+        },
+        JDM: {
+          ...aggsFilter,
+          aggs: { names: { terms: { field: "presenceJDM.keyword", missing: "NR", size: ES_NO_LIMIT } } },
+        },
+        depart: {
+          ...aggsFilter,
+          aggs: { names: { terms: { field: "departInform.keyword", size: ES_NO_LIMIT } } },
+        },
+        departMotif: {
+          ...aggsFilter,
+          aggs: { names: { terms: { field: "departSejourMotif.keyword", size: ES_NO_LIMIT } } },
+        },
+      },
+      size: 0,
+      track_total_hits: true,
+    };
+
+    return bodyYoung;
+  };
+  // création de la query pour récupèrer les infos des centres
+  const buildESRequestBodyForCohesion = (filters) => {
+    const bodyCohesion = {
+      query: {
+        bool: {
+          must: { match_all: {} },
+          filter: [
+            ...(filters.region?.length ? [{ terms: { "region.keyword": filters.region } }] : []),
+            ...(filters.department?.length ? [{ terms: { "department.keyword": filters.department } }] : []),
+            ...(filters.academy?.length ? [{ terms: { "academy.keyword": filters.academy } }] : []),
+            ...(filters.cohorts?.length ? [{ terms: { "cohorts.keyword": filters.cohorts } }] : []),
+          ].filter(Boolean),
+        },
+      },
+      aggs: {
+        typology: { terms: { field: "typology.keyword", size: ES_NO_LIMIT } },
+        domains: { terms: { field: "domain.keyword", size: ES_NO_LIMIT } },
+        capacity: { sum: { field: "placesTotal" } },
+      },
+      size: ES_NO_LIMIT,
+    };
+
+    return bodyCohesion;
+  };
+  // Création de la Query pour récupérer les info de la Session
+  const buildESRequestBodyForSession = (cohesionCenterId, filters) => {
+    const bodySession = {
+      query: {
+        bool: {
+          must: { match_all: {} },
+          filter: [{ terms: { cohesionCenterId } }, ...(filters.cohorts?.length ? [{ terms: { "cohort.keyword": filters.cohorts } }] : [])].filter(Boolean),
+        },
+      },
+      aggs: {
+        placesTotal: { sum: { field: "placesTotal" } },
+        placesLeft: { sum: { field: "placesLeft" } },
+        status: { terms: { field: "status.keyword" } },
+        timeSchedule: { terms: { field: "hasTimeSchedule.keyword" } },
+      },
+      size: ES_NO_LIMIT,
+    };
+
+    return bodySession;
+  };
+  // Creation de la Query pour récupérer les informations pour l'affichages des centres
+  const buildESRequestBodyForSessionCenter = (filters, sessionList) => {
+    const sessionPhase1Id = sessionList.map((session) => session._id).filter((id) => id);
+    const body = {
+      query: {
+        bool: {
+          must: { match_all: {} },
+          filter: [
+            ...(filters.status?.length ? [{ terms: { "status.keyword": filters.status } }] : []),
```

Comment:
```suggestion
           filters.status?.length ? { terms: { "status.keyword": filters.status } } : [],
```
Tu peux l'appliquer a tous